Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Forklift server Dockerfile
+# 2015-02-20
+
+# Pull base image
+FROM dockerfile/java:oracle-java8
+
+# Main developer
+MAINTAINER Matt Conroy <elduderino@mailinator.com>
+
+# Update system
+RUN apt-get update
+RUN apt-get upgrade -y
+
+# Install unzip
+RUN apt-get install unzip -y --force-yes --no-install-recommends
+
+# Add forklift server
+ADD ./server/target/universal/forklift-server-0.1.zip /tmp/forklift-server-0.1.zip
```

Comment:
Can you add a doc file explaining how the 0.1.zip file is generated? Or better yet, the process to follow to generate the docker container?